javascript/photo.js

```diff
@@ -167,6 +167,17 @@ backgrounds.Photo = new Model({
                     }
                 }
                 else {
+                    // Set the current background image first and then proceed with the caching the next images.
+                    var index = localStorage.getItem("index");
```

Comment:
% limitImages here.

We don't seem to be using "index" anywhere.

Reply:
To this and the rest of the comments -- this code definitely needs to be refactored and I'll take care of it in another commit (this weekend-ish).